tests/integration/test_io.py

```diff
@@ -17,6 +17,9 @@
 
 import numpy as np
 
+global CLEAN_NWB
```

Comment:
Is it necessary to make this a global variable? I see advantages to that. Note that this is effectively redefined in:
pynwb\tests\integration\ui_write\base.py:161 and
pynwb\tests\integration\ui_write\test_modular_storage.py:21

If we keep the global var here, can we use the same var in those places?

Reply:
I moved this to a reusable function in the subpackage pynwb.testing.